example/commands/regex_triggered.py

```diff
@@ -0,0 +1,10 @@
+from signalbot import Command, Context, regex_triggered
```

Comment:
Nice one but for consistency, can you also import it in the `bot.py` file in examples like the other example commands?